### ISSUE TITLE
fold, unfold the array

### ISSUE THREAD

**Issue by borodadada** [callumlocke/json-formatter#232]
I fold the array in Chrome, refresh the page and it is unfold. I need it to remember. The data is updated, and I have to hit minimize array every time.

**Comment by marcosox** [comment]
Yeah a couple more options would be great, like a preference to display collapsed/expanded by default, and a pair of "collapse all"/"expand all" buttons would impove this software by 1000%
